input/otlp/logs.go

```diff
@@ -109,12 +117,63 @@ func (c *Consumer) convertLogRecord(
 		}
 		event.Span.ID = spanID.HexString()
 	}
-	if attrs := record.Attributes(); attrs.Len() > 0 {
+	attrs := record.Attributes()
+	if isError(record) {
+		event.Processor = model.ErrorProcessor
+		setErrorLabels(attrs, &event)
+		updateErrorType(attrs, &event)
+	} else {
 		setLabels(attrs, &event)
 	}
 	return event
 }
 
+func updateErrorType(m pcommon.Map, event *model.APMEvent) {
+	event.Event.Kind = "event"
+	event.Event.Category = "device"
```

Comment:
Should the category always be set, even if it is a _regular_ log exception? My understanding is that we always want to translate to error events if `semconv.AttributeExceptionType` or `semconv.AttributeExceptionMessage` is set, but only label as app crash (`event.name=crash` & `event.category=device`) when the custom `event.name=crash` is set.